src/desktop/apps/artwork/client/errorModal.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import { ErrorModal } from "@artsy/reaction/dist/Components/Modal/ErrorModal"
```

Comment:
This can be aliased to `import { ErrorModal } from  "reaction/Components/..."`

Reply:
My editor doesn't like that 🙁

Reply:
hmm 🤔 

<img width="472" alt="screen shot 2018-10-01 at 3 56 58 pm" src="https://user-images.githubusercontent.com/236943/46320342-b35e5100-c592-11e8-95ce-16a688f5db9e.png">

<img width="710" alt="screen shot 2018-10-01 at 3 56 42 pm" src="https://user-images.githubusercontent.com/236943/46320343-b3f6e780-c592-11e8-9f1f-e62d57c546a6.png">

maybe try to reload editor?